src/Kiryuu/KiryuuParser.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+import { MangaStreamParser } from "../MangaStreamParser";
```

Comment:
Reason for custom parser?

Reply:
As you can see, the "Author", "Artists" things won't show. I need to use this.
Also i need to trim the Titles.